### PR TITLE
Add ProtocolSupport plugin

### PR DESCRIPTION
**Pull request by DNx5**
added tag for player's protocol version.
Resource Page: http://www.spigotmc.org/resources/protocolsupport.2367/

### CODE REVIEW

**Comment by mcmonkey4eva** [comment]
... You've never used Github before, have you?

**Comment by mcmonkey4eva** [comment]
Also, why should we link this plugin and not just copy over the code for this one tag you want?

**Comment by mcmonkey4eva** [comment]
Nevermind, misunderstood the purpose of the plugin

File: src/main/java/net/gnomeffinway/depenizen/extensions/protocolsupport/ProtocolSupportExtension.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        // @plugin Depenizen, ProtocolSupport
+        // -->
+        if (attribute.startsWith("version")) {
+            if(player == null)
+                return null;
+
+            attribute = attribute.fulfill(1);
+
+            // <--[tag]
+            // @attribute <p@player.version>
```
Comment (Morphan1) [inline src/main/java/net/gnomeffinway/depenizen/extensions/protocolsupport/ProtocolSupportExtension.java]: Shouldn't this be `<p@player.version.protocol>`?
Also, would it make more sense to make it `<p@player.protocol>` and `<p@player.protocol.id>`?

Comment (DNx5) [inline src/main/java/net/gnomeffinway/depenizen/extensions/protocolsupport/ProtocolSupportExtension.java]: I didn't notice that.
I don't know, but <p@player.version> will return the client version based on protocol version, example. MINECRAFT_1_8
and <p@player.version.protocol> will return the protocol version, like 47 for 1.8.x client.

**Comment by mcmonkey4eva** [comment]
@DNx5 pls...

**Pull request closed without merging**